client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -172,12 +172,6 @@ export class CommentNavigation extends Component {
 					) }
 				</NavTabs>
 
-				<CommentNavigationTab className="comment-navigation__actions comment-navigation__open-bulk">
```

Comment:
Haha, either way. We'll be working on this in the short term so I'm flexible on options here.